benchmark/atoi-corpus.h

```diff
@@ -119,17 +119,28 @@ struct CorpusStringLength {
     }
 };
 
+
+#if defined(__AVX2__)
+#define AVX2_STRLEN_CORPUS_X_LIST \
+  X(ZOO_AVX, zoo::avx2_strlen)
```

Comment:
Nit: please 4 space indentation?

Not just that I dislike inconsistent indentation, but that it "clogs the flow" of most programmers. I honestly don't know if it is my subjective dislike or the objective problem of accidental levels of indentation what bothers me.

Thanks to this presenation by Kevlin Henney, which I think is a true classic of our discipline (time marked to make it more relevant to this comment),
https://youtu.be/oyyFKHpzL0Q&t=2217
I learned the problem of the accidental levels of indentation, in this case in particular, using 4 spaces (or something equivalent, all around) is the only way to convey "these spaces only mean one level of detail deeper", the use of two spaces and 4 spaces in other places begs the question of whether the two spaces have their own alternative meaning, etc.

Reply:
Thanks for this. Habits from working in a two space cpp codebase.